pages/setup/index.vue

```diff
@@ -108,15 +159,27 @@ export default {
                         voters: this.voters,
                     })
                 } catch (e) {
+                    // eslint-disable-next-line no-console
                     console.error(e);
                 }
             } catch (err) {
+                // eslint-disable-next-line no-console
                 console.error(err);
                 this.error = err;
             } finally {
                 this.isLoading = false;
             }
         },
+        voterListRule(list){
+            if(list.length===0) return 'There must be at least one voter.'
```

Comment:
```suggestion
            if (list.length===0) return 'There must be at least one voter.'
```
Not sure if linter is working correctly, can you try running `lint` in npm and possibly `--fix`?

Reply:
Eslint did not point out this. I will just add the space for now.